src/components/subscriptions/licenses/LicenseManagementTable/LicenseManagementTableBulkActions.jsx

```diff
@@ -181,6 +184,70 @@ const LicenseManagementTableBulkActions = ({
     );
   };
 
+  const getTotalToRevoke = () => {
```

Comment:
nit: This could likely utilize `useMemo` to only compute the value when necessary rather the re-calculate it on each render.